pkg/api/config.go

```diff
@@ -27,10 +27,9 @@ type Config struct {
 	Images       ImageConfig       `json:"images,omitempty"`
 
 	// kubeconfigs
-	AdminKubeconfig              *v1.Config `json:"adminKubeconfig,omitempty"`
-	MasterKubeconfig             *v1.Config `json:"masterKubeconfig,omitempty"`
-	NodeBootstrapKubeconfig      *v1.Config `json:"nodeBootstrapKubeconfig,omitempty"`
-	AzureClusterReaderKubeconfig *v1.Config `json:"azureClusterReaderKubeconfig,omitempty"`
```

Comment:
so we had no use case for AzureClusterReader?

Reply:
What will blackbox monitoring will use?

Reply:
grr, that is a good question @mjudeikis, let's talk to msft about that today

Reply:
Alright, repurposing AzureClusterReader to BlackBoxMonitor.  Stopped using AzureClusterReader in tests.